project/Boilerplate.scala

```diff
@@ -472,6 +475,11 @@ object Boilerplate {
         else
           s"def parMapN[Z](f: (${`A..N`}) => Z)(implicit p: NonEmptyParallel[M]): M[Z] = Parallel.parMap$arity($tupleArgs)(f)"
 
+      val parFlatMap =
+        if (arity == 1)
+          s"def parFlatMap[Z](f: (${`A..N`}) => M[Z])(implicit p: NonEmptyParallel[M]): M[Z] = p.flatMap.flatMap($tupleArgs)(f)"
```

Comment:
there is not `par` here, right? This is just flatMap isn't it? Do we really need arity 1?

Reply:
True. It already existed for `parMap` above. I think it's fine, there's nothing fundamentally wrong with it.

Reply:
I think that there's nothing fundamentally wrong too. Plus I prefer (and as a user expect) consistency with its brother `parMap` that has the arity one method.